src/layers/src/Pane.js

```diff
@@ -32,90 +79,26 @@ const Pane = memo(
     } = props
     const theme = useTheme()
 
-    function getHoverElevationStyle(hoverElevation, css) {
-      if (!Number.isInteger(hoverElevation)) return {}
-
-      return {
-        transitionDuration: '150ms',
-        transitionProperty: 'box-shadow, transform',
-        transitionTimingFunction: `cubic-bezier(0.0, 0.0, 0.2, 1)`,
-        ':hover': {
-          ...(css[':hover'] || {}),
-          transform: 'translateY(-2px)',
-          boxShadow: theme.elevations[hoverElevation]
-        }
-      }
+    const styles = {
+      ...css,
```

Comment:
Oof - while we're here, can we yank this out? 😅

Reply:
Unfortunately no, because of the other internal components that rely on it.

Reply:
Roger that - just noticed `<CornerDialog />`, `<Dialog />`, and `<SideSheet />` are still using it